File: rdb/src/main/java/com/sequenceiq/rdb/service/registry/ServiceAddressResolvingException.java
package com.sequenceiq.rdb.service.registry;

public class ServiceAddressResolvingException extends Exception {
    public ServiceAddressResolvingException(String message) {
        super(message);
    }

    public ServiceAddressResolvingException(String message, Throwable cause) {
        super(message, cause);
    }

    public ServiceAddressResolvingException(Throwable cause) {
        super(cause);
    }
}

File: autoscale/src/main/java/com/sequenceiq/periscope/service/registry/ServiceAddressResolvingException.kt
package com.sequenceiq.periscope.service.registry

class ServiceAddressResolvingException : Exception {
    constructor(message: String) : super(message) {
    }

    constructor(message: String, cause: Throwable) : super(message, cause) {
    }

    constructor(cause: Throwable) : super(cause) {
    }
}
